sherpa/astro/utils/tests/test_astro_utils_xspec.py

```diff
@@ -332,8 +333,82 @@ def test_parse_multiplicative_model():
     assert model.pars[3].hardmax == pytest.approx(1e25)
 
 
+def check_compiled(got, suffix):
+    """Check the compiled output.
 
-def test_create_model_additive():
+    The output depends on the XSPEC version because it will cause
+
+        #define XSPEC_major_micro_minor
```

Comment:
Typically, versions are defined in major.minor.micro. Is the order in XSPEC_major_micro_minor correct? (Comment applies to other references in this file)

Reply:
As mentioned this was a thinko on my part - see my comment at https://github.com/sherpa/sherpa/pull/1730#discussion_r1332967520